### PR TITLE
Added options startingX and startingY in Add-Qr Module

### PR DESCRIPTION
**Pull request by root00198**
Added startingX and startingY for the QR Code Module.

output

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by root00198** [comment]
@publiclab/is-reviewers

File: src/modules/AddQR/info.json [hunk 1]
```diff
@@ -12,6 +12,16 @@
         "type": "string",
         "desc": "input string to generate QR code",
         "default": "https://github.com/publiclab/image-sequencer"
+      },
+      "startingX": {
+        "type": "integer",
```
Comment (HarshKhandeparkar) [inline src/modules/AddQR/info.json]: Why not make these strings and use the util function `parseCornerCoordinateInput`? That will allow the user to input values like `50%` as well as `50`

Comment (root00198) [reply src/modules/AddQR/info.json]: done

**Comment by HarshKhandeparkar** [comment]
Waiting for the tests...

**Comment by root00198** [comment]
@blurry-x-face, technically it should be generated outside the image(means no image should be generated), and for loop will restrict the creation of image in that case.

**Comment by blurry-x-face** [comment]
> @blurry-x-face, technically it should be generated outside the image(means no qr code image should be generated), and for loop will restrict the creation of QR code in that case.

In that case maybe we can just generate the QR at the corner? What do you think?

**Comment by root00198** [comment]
The fact is that maybe someone wants only half a qr code, I feel like by not sticking the qr at the end, we are giving a feature to someone.

**Comment by root00198** [comment]
@blurry-x-face  what do you say?

**Comment by blurry-x-face** [comment]
Why would someone want only half a QR code? what are your thoughts @HarshKhandeparkar

**Comment by root00198** [comment]
maybe not half a qr code, maybe he/she can try to minimize the outer white area of the qr code. by putting it a bit off.

**Comment by root00198** [comment]
@blurry-x-face, now I feels like you are right. Maybe fixing the qr code at the end is better.

**Comment by root00198** [comment]
Screenshot from 2020-01-18 13-33-16

**Comment by jywarren** [comment]
Really nice! Thank you!

**Pull request merged**